Sources/Networking/Responses/OfferingsResponse.swift

```diff
@@ -54,7 +54,62 @@ struct OfferingsResponse {
     let placements: Placements?
     let targeting: Targeting?
     let uiConfig: UIConfig?
+    let config: Config?
```

Comment:
Nit: we should probably name this something more specific, like `priceFormattingConfig`, in case we decide to add other configs in the future

Reply:
Good one, I figured we might want to use this as a 'config envelope' to also hold future configuration values, in order to avoid having to update all initializers and mocks again. But I will make sure to bring this up in my PR and ask for the team's opinion on this as well